Pod/Classes/Logo/UIView+Logo.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UIView+Logo.swift
+//  Pods
+//
+//  Created by Derek Ostrander on 7/18/16.
+//
+//
+
+import UIKit
+
+public extension UIView {
+    private class RZBundle {} // to grab the image url
+    public static func rz_logo() -> UIImageView {
+        guard let url = NSBundle(forClass: RZBundle.self).URLForResource("logo-built-by-RZ", withExtension: "png"),
```

Comment:
Reason why I didn't before was just I was using the older ones. PDF i say is the way to go.